contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/HttpScanBatchCreator.java

```diff
@@ -99,6 +99,8 @@ public void bind(ManagedScanFramework framework) { }
       if (count++ == 0) {
         if (inputType.equalsIgnoreCase("csv")) {
           return new HttpCSVBatchReader(subScan);
+        } else if (inputType.equalsIgnoreCase("xml")) {
```

Comment:
Could 'csv' and 'xml' reuse the final static variables recommended above?

Reply:
Fixed